src/trunk.h

```diff
@@ -396,6 +396,13 @@ trunk_reset_stats(trunk_handle *spl);
 
 void
 trunk_print(platform_log_handle *log_handle, trunk_handle *spl);
+
+void
+trunk_print_super_block(platform_log_handle *log_handle, trunk_handle *spl);
+
+void
+trunk_sprint(platform_stream_handle user_stream, trunk_handle *spl);
+
```

Comment:
Again wasn't this going to change?

Reply:
Straggling reference from previous version. Deleted. [ Sorry ! ]